README.md

```diff
@@ -10,6 +10,7 @@ When you want slack to turn into a tire fire while also being productive at the
 
 It's ezpz. You can look at the example implementation [here](https://github.com/konecnyna/slack-cat/tree/master/example).
 
+0. Recommended Node Version: `lts/carbon` (v8.11)
```

Comment:
This should prob be 10+ . I AINT SUPPORTING OLD NODES!!!!!!!!!!! 😄

Reply:
Using stable `v10.0` I received the following error attempting to `npm install`

```
Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167░░░░░░⸩ ⠏ extract:moment: timing audit body Completed in 1ms
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
npm WARN slack-cat-test@1.0.0 No description
npm WARN slack-cat-test@1.0.0 No repository field.

npm ERR! cancel after 1 retries!
WARN tar ENOENT: no such file or directory, open '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects/mssql/resource-lock.js'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/dialects'
WARN tar ENOENT: no such file or directory, open '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/errors.js'
WARN tar ENOENT: no such file or directory, lstat '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/errors'
WARN tar ENOENT: no such file or directory, open '/Users/ahmadragab/Documents/Dev/slack-cat-test/node_modules/.staging/sequelize-b5faeccf/lib/hooks.js'
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
events.js:167
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_stream_writable.js:243:12)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    at PassThrough.Writable.end (_stream_writable.js:592:10)
    at ReadEntry.entry.on (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/pacote/lib/extract-stream.js:19:41)
    at ReadEntry.emit (events.js:187:15)
    at ReadEntry.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at ReadEntry.[maybeEmitEnd] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:249:12)
    at ReadEntry.end (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:162:27)
    at Unpack.[consumeBody] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:210:13)
    at Unpack.[consumeChunkSub] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:391:40)
    at Unpack.[consumeChunk] (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:362:30)
    at Unzip.(anonymous function).on.chunk (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/lib/parse.js:291:59)
    at Unzip.emit (events.js:182:13)
    at Unzip.emit (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:296:25)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minipass/index.js:99:17)
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
Emitted 'error' event at:
    at writeAfterEnd (_stream_writable.js:245:10)
    at PassThrough.Writable.write (_stream_writable.js:292:5)
    [... lines matching original stack trace ...]
    at Unzip.write (/Users/ahmadragab/.nvm/versions/node/v10.0.0/lib/node_modules/npm/node_modules/tar/node_modules/minizlib/index.js:284:29)
```